db/pebble/transaction.go

```diff
@@ -94,7 +94,7 @@ func (t *Transaction) Get(key []byte, cb func([]byte) error) error {
 		return ErrDiscardedTransaction
 	}
 
-	defer t.listener.OnIO(false, time.Since(start)) //nolint:govet
```

Comment:
it was intentional: https://github.com/NethermindEth/juno/pull/1430/files#r1395294500 
Let's add comment why we're doing it this way (see comment in the link)